code/game/dna/genes/disabilities.dm

```diff
@@ -93,8 +93,8 @@
 		block=TWITCHBLOCK
 
 /datum/dna/gene/disability/nervousness
```

Comment:
Change the names in the paths aswell and all references to it.

Reply:
Is this really needed? I deliberately avoided this to make as less of an impact as possible.

Reply:
Yes, that is needed to avoid confusions in the future.

Reply:
Done now, hasn't caused runtime issues in testing